react-components/src/metadata/admin/EditAttributeFormList.js

```diff
@@ -223,9 +233,12 @@ class EditAttribute extends Component {
                                 </ExpansionPanelSummary>
                                 <ExpansionPanelDetails>
                                     <FieldArray name={`${field}.values`}
-                                                component={AttributeEnumEditGrid}
-                                                parentID={formID}
-                                                change={change}
+                                                render={arrayHelpers => (
+                                                    <AttributeEnumEditGrid
+                                                        {...arrayHelpers}
+                                                        parentID={formID}
```

Comment:
Unfortunately Formik's `FieldArray` won't pass extra props down to its given `component`, so in order to pass things like this `parentID` prop to the inner `component`, I had to use the FieldArray's `render` prop instead of simply `component`.

Reply:
👍